ui/component/claimPreview/view.jsx

```diff
@@ -195,31 +206,37 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
   }
 
   useEffect(() => {
-    if (isValid && !isResolvingUri && shouldFetch && uri) {
```

Comment:
Why remove the check for `isResolvingUri`?

Reply:
It caused a weird infinite loop. I'll try again with it in..

Reply:
Apparently unable to reproduce - I put it back.